docs/news.rst

```diff
@@ -576,19 +598,20 @@ few parsers utilized it now. All available roles can be listed with
     DTS	d	undef	undefined	on
     DTS	h	system	system header	on
     DTS	h	local	local header	on
-    Make	I	generic	non-categorized generic role	on
-    Make	I	optional	included as an optional makefile	on
-    Sh	s	generic	non-categorized generic role	on
+    Make	I	included	included	on
+    Make	I	optional	optionally included	on
+    Sh	s	loaded	loaded	on
     Vera	d	undef	undefined	on
```

Comment:
That's obviously still only a partial list. It might be worth making that clear.

I didn't update it to the full list as it seemed a bit long now.